extensions/wikia/UserLogin/tests/UserSignupTest.php

```diff
@@ -268,9 +268,9 @@ public function signupDataProvider() {
 				'password' => '123',
 			);
 
-			$reqParams18 = array(
-				'userloginext01' => self::TEST_USERNAME . ':test' ,
-			);
+			$reqParams18 = [
```

Comment:
minor:
If we change array to `[]` change it in whole file, or don't change it and keep `file` convention